tests/core/tests/resources.py

```diff
@@ -1349,7 +1349,7 @@ class Meta:
 
 class ModelResourceTestCase(TestCase):
     fixtures = ['note_testdata.json']
-    urls = 'core.tests.field_urls'
+    urls = 'core.tests.resource_urls'
```

Comment:
I think that because of this line, core.tests.field_urls now has no coverage, which is why the percentage dropped.

core.tests.field_urls is referenced here too: https://coveralls.io/files/1857577512#L955
But it doesn't look like it uses the test client at all.

Since it appears field_urls.py isn't actually needed, I would delete it, and then delete this line:
https://coveralls.io/files/1857577512#L957

Don't forget to add this PR to the list in docs/release_notes/dev/rst. Then squash your commits.